settings.gradle

```
@@ -1,3 +1,2 @@
-rootProject.name = 'RoboViz'
+rootProject.name = 'RoboViz-Kotlin'
```

```suggestion
rootProject.name = 'RoboViz'
```
We should probably rename the root project back to RoboViz.